Svg.Editor.Core/SvgDrawingCanvas.cs

```diff
@@ -361,7 +367,13 @@ public async Task OnDraw(IRenderer renderer)
 			}
         }
 
-		private void ApplyConstraintsFillUniform()
+        private void OnInitialDraw(IRenderer obj)
+        {
```

Comment:
Why here? Does that make sense or are you just playing around?

Reply:
Yeah i forgot the event keyword. So i guess i have to invoke on add and this was here because the action would be null